views/js/qtiCreator/helper/saveChanges.js

```diff
@@ -27,6 +29,9 @@ define(function () {
      * @returns {Promise}
      */
     return itemCreator => new Promise((resolve, reject) => {
+        //trigger event to restore all currently active widget back to sleep state
```

Comment:
do you consider the comment is required ?

Reply:
I think it can make it clearer as it was done [here](https://github.com/oat-sa/extension-tao-itemqti/blob/1d40adc6e084aba20227efef331f3abd7e701a15/views/js/qtiCreator/editor/blockAdder/blockAdder.js#L48)